packages/cardano-graphql-services/src/TxSubmit/TxSubmitHttpServer.ts

```diff
@@ -0,0 +1,53 @@
+import { HttpServer, ListenConfig } from '../Http';
+import { Logger, dummyLogger } from 'ts-log';
+import { TxSubmitProvider } from '@cardano-sdk/core';
+import express, { Router } from 'express';
+const bodyParser = require('body-parser');
+
+export interface TxSubmitServerDependencies {
+  txSubmitProvider: TxSubmitProvider;
+  logger?: Logger;
+}
+
+export class TxSubmitHttpServer extends HttpServer {
+  private constructor(config: ListenConfig, router: Router, logger = dummyLogger) {
+    super({ listen: config, name: 'TxSubmitServer', router }, logger);
+  }
+  static create(config: ListenConfig, { txSubmitProvider, logger = dummyLogger }: TxSubmitServerDependencies) {
+    const router = express.Router();
+    // Review: Can we set this limit dynamically?
```

Comment:
Could come from env/cli variable. Or are you considering getting it from protocol parameters?

Reply:
Protocol params, but I guess config would be a minimum.